spec/features/posts_can_have_a_newline_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
+require 'date'
+
+RSpec.feature "Format", type: :feature do
+  scenario "User can use newlines in posts" do
+    visit "/posts"
+    click_link "New post"
+    fill_in "Message", with: "Hello, world!\nHello back at ya!"
+    click_button "Submit"
+    expect(page).to have_content("Hello, world!\nHello back at ya! - #{Date.today.to_s}")
```

Comment:
Just a question - when I run on a server I see that the date appears on its own line and not encased in any <p>
One we may need to revisit when it comes to styling later.